### DISCUSSION TITLE
Invisible links in mermaid

### DISCUSSION THREAD

**Discussion by aghaynes** [quarto-dev/quarto-cli#3349] [Q&A] [answered]
According to the help files for mermaid, it is possible to have invisible links between nodes in mermaid charts, which can be particularly useful for enforcing a particular layout. https://mermaid-js.github.io/mermaid/#/flowchart?id=an-invisisble-link E.g. the following code should produce the flowchart further down.

````
```{mermaid}
flowchart LR
  box1 --> box2
  box2 --- box3
  box4 ~~~ box2
```
````

![grafik](https://user-images.githubusercontent.com/6594028/202184980-eb80dff4-305e-4c94-96c5-fcf78c9f62bb.png)

Executing the chunk in quarto produces a syntax error though. 

It seems like the invisible links are probably a new feature (there's an issue referencing a commit in mid-October - https://github.com/mermaid-js/mermaid/issues/3192). 

I guess this would require an update to the version of mermaid in quarto?

**Comment by cscheid** [comment, marked as answer]
Yes, it would require an update. We'll do it in v1.3, likely later today.
